src/HotChocolate/Core/src/Types/Types/Introspection/__Field.cs

```diff
@@ -60,8 +73,12 @@ public static string Name(IPureResolverContext context)
         public static string? Description(IPureResolverContext context)
             => context.Parent<IOutputField>().Description;
 
-        public static IFieldCollection<IInputField> Arguments(IPureResolverContext context)
-            => context.Parent<IOutputField>().Arguments;
+        public static object? Arguments(IPureResolverContext context)
+            => context.Parent<IOutputField>() is { } of
```

Comment:
can this ever be null?

original expression was `context.Parent<IOutputField>().Arguments;`

the is check is not needed in this case

```
{
    IOutputFIeld field =  context.Parent<IOutputField>() ;
    return context.ArgumentValue<bool>(Names.IncludeDeprecated)
                     ? field.Arguments
                     : field.Arguments.Where(t => !t.IsDeprecated);
}
```